### DISCUSSION TITLE
[QUESTION] Active Maintenance for Longhorn 1.4

### DISCUSSION THREAD

**Discussion by davidmateos** [longhorn/longhorn#8062] [General]
## Question
According to https://github.com/longhorn/longhorn#releases, 1.4, 1.5, and 1.6 releases are under Active Maintenance. 
When will 1.4 cease to be maintained?
Is there a policy on the maintenance of releases? (e.g. only the last 3 releases are maintained)

**Comment by innobead** [comment]
> ## Question
> According to [longhorn/longhorn#releases](https://github.com/longhorn/longhorn#releases), 1.4, 1.5, and 1.6 releases are under Active Maintenance. When will 1.4 cease to be maintained? Is there a policy on the maintenance of releases? (e.g. only the last 3 releases are maintained)

It's one year since the first stable version of a release branch. For 1.4. the first release is 1.4.1 (March 2023), so the EOL is March 2024. 

Suggest checking https://github.com/longhorn/longhorn/wiki/Release-Schedule-&-Support#release-support.

**Reply by davidmateos** [reply]
Gotcha. Thanks @innobead
